lib/phoenix/controller.ex

```diff
@@ -704,4 +704,16 @@ defmodule Phoenix.Controller do
   defp persist_flash(conn, value) do
     put_private(conn, :phoenix_flash, value)
   end
+
+  @doc """
+  Invokes the controller plug and calls its action
+
+  ## Examples
+
+      iex> Controller.call_action(conn, MyApp.MyController, :index)
+
+  """
+  def call_action(conn, controller, action) do
```

Comment:
I agree. We just had a bit of duplication, but it's hardly any work to invoke ourselves, so I'll remove.